app/registries/foreman/access_control.rb

```diff
@@ -100,6 +122,10 @@ def security_block(name, options = {})
       def mapped_permissions
         @permissions
       end
+
+      def actions_to_add
+        @added_permission_actions.flatten(0)
```

Comment:
I needed to read into the docs to find out what this does.
I think this only does `to_a` and I do not get why, we can iterate over the hash directly